packages/performance-tests/test/TransformerRegression.test.ts

```diff
@@ -0,0 +1,135 @@
+    Logger.initializeToConsole();
+    Logger.setLevelDefault(LogLevel.Error);
+    Logger.setLevel(loggerCategory, LogLevel.Info);
+    Logger.setLevel(TransformerLoggerCategory.IModelExporter, logLevel);
+    Logger.setLevel(TransformerLoggerCategory.IModelImporter, logLevel);
+    Logger.setLevel(TransformerLoggerCategory.IModelTransformer, logLevel);
+  }
+  let usrEmail;
+  let usrPass;
+  if(process.env.V2_CHECKPOINT_USER_NAME){
+    usrEmail = process.env.V2_CHECKPOINT_USER_NAME;
+    usrPass = process.env.V2_CHECKPOINT_USER_PASSWORD;
+  } else if(process.env.V1_CHECKPOINT_USER_NAME){
+    usrEmail = process.env.V1_CHECKPOINT_USER_NAME;
+    usrPass = process.env.V1_CHECKPOINT_USER_PASSWORD;
+  } else {
+    usrEmail = process.env.IMODEL_USER_NAME;
+    usrPass = process.env.IMODEL_USER_PASSWORD;
+  }
+
+  assert(usrEmail, "user name was not configured");
+  assert(usrPass, "user password was not configured");
+  const user = {
+    email: usrEmail,
+    password: usrPass,
+  };
+
+  assert(process.env.OIDC_CLIENT_ID, "");
+  assert(process.env.OIDC_REDIRECT, "");
+  assert(process.env.IMJS_URL_PREFIX, "")
+  assert(process.env.OIDC_SCOPE, "List scopes");
+  const authClient = process.env.CI === "1"
+    ? new TestBrowserAuthorizationClient({
+      clientId: process.env.OIDC_CLIENT_ID,
+      redirectUri: process.env.OIDC_REDIRECT,
+      scope: process.env.OIDC_SCOPE,
+      authority: `https://${process.env.IMJS_URL_PREFIX}ims.bentley.com`,
+    }, user)
+    : new NodeCliAuthorizationClient({
+      clientId: process.env.OIDC_CLIENT_ID,
+      redirectUri: process.env.OIDC_REDIRECT,
+      scope: process.env.OIDC_SCOPE,
+    });
+
+  await authClient.signIn();
+
+  const hostConfig  = new IModelHostConfiguration();
+  hostConfig.authorizationClient = authClient;
+  const hubClient = new IModelsClient({ api: { baseUrl: `https://${process.env.IMJS_URL_PREFIX}api.bentley.com/imodels` } });
+  hostConfig.hubAccess = new BackendIModelsAccess(hubClient);
+  await IModelHost.startup(hostConfig);
+
+  return realizeAsyncIterator(getTestIModels());
+};
+
+// Mocha tests must know the test cases ahead time, so we collect the the Imodels first before beginning the tests
+async function realizeAsyncIterator(getTestIModels:AsyncGenerator<TestIModel>) {
+  let arrImodels:TestIModel[] = [];
+  const iModelIdStr = process.env.IMODEL_IDS;
+  assert(iModelIdStr, "no Imodel Ids")
+  const iModelIds = iModelIdStr === "*" ? "" : iModelIdStr.split(",")
+  for await (const iModel of getTestIModels) {
+    if(iModelIds.includes(iModel.iModelId) || iModelIds === "" )
+      arrImodels.push(iModel);
+  }
+  return arrImodels;
+}
+
+async function runRegressionTests() {
+  const testIModels = await setupTestData();
+  var reporter = new Reporter();
+  const reportPath = initOutputFile("report.csv", outputDir);
+
+  describe('Transformer Regression Tests', function () {
+    testIModels.forEach(async (value) => {
+      const iModel = value;
```

Comment:
is this alias really necessary? The closure passed to `it` should be able to access that value. It's only `for (let blah of)` that has this issue iirc... a `forEach` shouldn't have this problem at all, and neither should a `for (const _ of)` loop...

Reply:
done